cmd/node/apis/types.go

```diff
@@ -13,3 +13,7 @@ type msgReq struct {
 	ProjectVersion string `json:"projectVersion"   binding:"required"`
 	Data           string `json:"data"             binding:"required"`
 }
+
+type HandleRsp struct {
```

Comment:
rename to msgResp ? for alignment with msgReq

Reply:
this type was exported, and used by other packages